src/databricks/labs/ucx/azure/access.py

```diff
@@ -31,6 +31,7 @@ class StoragePermissionMapping:
     principal: str
     privilege: str
     type: str
+    default_network_action: str = "Unknown"  # "Deny", "Allow" or "Unknown"
```

Comment:
@nkvuong : It's confirmed that with a default we can read csv's missing this column. Could you please review the PR?